index.js

```diff
@@ -102,7 +95,11 @@ function mdToHtml (filePath, githubHeadings) {
     .use(markdownItMark)
     .use(markdownItAbbr)
 
-  if (githubHeadings) {
+  if (opts.hljsHighlights) {
+    md = md.use(makdowniItHighlightjs, {auto: false, code: true})
```

Comment:
This should have been `.use(makdowniItHighlightjs, {auto: false, code: opts.hljsHighlights})`